project/Dependencies.scala

```diff
@@ -36,7 +36,7 @@ object Dependencies {
     // If changing this, remember to update protoc plugin version to align in
     // maven-plugin/src/main/maven/plugin.xml and org.apache.pekko.grpc.sbt.PekkoGrpcPlugin
     val googleProtoc = "3.20.1" // checked synced by VersionSyncCheckPlugin
```

Comment:
Can this two version be the same?

Reply:
I guess we should also use a version of protoc that doesn't have vulnerabilities but this is maybe for 1.1.x.

Reply:
the vulnerabilities are in protobuf-java, not in protoc